internal/flex/flex.go

```diff
@@ -23,13 +23,27 @@ const (
 func ExpandStringList(configured []interface{}) []*string {
 	vs := make([]*string, 0, len(configured))
 	for _, v := range configured {
-		if v, ok := v.(string); ok && v != "" {
+		if v, ok := v.(string); ok && v != "" { // v != "" is not necessary here, "" = !ok (weird Go-ism)
 			vs = append(vs, aws.String(v))
 		}
 	}
 	return vs
 }
 
+// ExpandStringListEmpty the result of flatmap. Expand for an array of strings
+// and returns a []*string. Empty strings are NOT skipped.
+func ExpandStringListEmpty(configured []interface{}) []*string {
```

Comment:
Add unit test(s) to `internal/flex/flex_test.go`?

Reply:
Yeah, good idea. The unit tests can easily give a false positive if we put `""` (which is not what Terraform does) instead of `nil` in the test.